src/core/ready.js

```diff
@@ -1,6 +1,7 @@
 define( [
 	"../core",
 	"../var/document",
+	"../core/readyException",
```

Comment:
I'll submit a docs PR once this is accepted, no worries.

At the meeting we agreed that since this issue kind of breaks all Angular apps (as they wrap their init code in `$()` and all bootstrapping errors are silenced) we'll release a quick 3.1.0 with maybe only this fix. I renamed previous milestones: `3.0.1` to `3.1.1` and `3.1.0` to `3.2.0`.